workflow/rules/common.smk

```diff
@@ -239,7 +239,10 @@ def get_recalibrate_quality_input(wildcards, bai=False):
     elif is_activated("remove_duplicates"):
         return "results/dedup/{{sample}}.{ext}".format(ext=ext)
     else:
-        return "results/mapped/bwa/{{sample}}.{ext}".format(ext=ext)
+        if is_activated("ref/pangenome"):
+            return "results/mapped/vg/{{sample}}_rg_added.{ext}".format(ext=ext)
+        else:
+            return "results/mapped/bwa/{{sample}}.{ext}".format(ext=ext)
```

Comment:
Use f-string in both cases, use `{mapper}` or something like that in order to not mostly repeat the f-string.